docs/deploying-workloads.md

```diff
@@ -0,0 +1,91 @@
+# Deploying workloads on registered edge devices
+
+Workload can be deployed to devices __after__ they are registered with the cluster.
+
+`EdgeDeployment` can be deployed only on edge devices in the same namespace.
+
+
+## Deploying by device ID
+
+---
+**Note**
+
+When both methods are used in one `EdgeDeployment`, this one takes precedence over the selector-based one.
```

Comment:
what do you think of preventing both options and changing the CRD validation to oneOf to enforce it ?

Reply:
The CRDs are generated from sources and I don't think that controller-gen supports generating `oneOf`. I'll try to find out.